src/vendor/wordpress/styles/_core-blocks.scss

```diff
@@ -9,6 +9,7 @@
 @use '../../../mixins/ms';
 @use '../../../mixins/table';
 @use '../../../mixins/theme';
+@use '../../../objects/container/container';
```

Comment:
Won't this result in the container CSS being output twice (once in its original position, and again here)?

It might be a better idea to use design tokens to share properties.

Reply:
The Sass documentation says it'll only be rendered once: https://sass-lang.com/documentation/at-rules/use

And if you review the compiled file in the `/dist` folder, you'll only see it once.

That said, I think you're right. What I'm doing is violating the intent of our design tokens. I'll see if I can refactor those into tokens.